memcached.c

```diff
@@ -2357,7 +2357,7 @@ enum delta_result_type do_add_delta(LIBEVENT_THREAD *t, const char *key, const s
         }
         memcpy(ITEM_data(new_it), buf, res);
         memcpy(ITEM_data(new_it) + res, "\r\n", 2);
-        item_replace(it, new_it, hv);
+        item_replace(it, new_it, hv, (settings.use_cas) ? get_cas_id() : 0);
```

Comment:
This has to be wrong.

Reply:
nevermind. it's being swapped out by the caller... which is fine since it's not actually used anywhere else.